docs/docs/install/environment-variables.md

```diff
@@ -183,13 +183,10 @@ Other machine learning parameters can be tuned from the admin UI.
 
 ## Prometheus
 
-| Variable                       | Description                                                                                   | Default | Containers | Workers            |
-| :----------------------------- | :-------------------------------------------------------------------------------------------- | :-----: | :--------- | :----------------- |
-| `IMMICH_METRICS`<sup>\*1</sup> | Toggle all metrics (one of [`true`, `false`])                                                 |         | server     | api, microservices |
-| `IMMICH_API_METRICS`           | Toggle metrics for endpoints and response times (one of [`true`, `false`])                    |         | server     | api, microservices |
-| `IMMICH_HOST_METRICS`          | Toggle metrics for CPU and memory utilization for host and process (one of [`true`, `false`]) |         | server     | api, microservices |
-| `IMMICH_IO_METRICS`            | Toggle metrics for database queries, image processing, etc. (one of [`true`, `false`])        |         | server     | api, microservices |
-| `IMMICH_JOB_METRICS`           | Toggle metrics for jobs and queues (one of [`true`, `false`])                                 |         | server     | api, microservices |
+| Variable                   | Description                                                                  | Default | Containers | Workers            |
+| :------------------------- | :--------------------------------------------------------------------------- | :-----: | :--------- | :----------------- |
+| `IMMICH_TELEMETRY_INCLUDE` | Collect these telemetries. List of `all`, `host`, `api`, `io`, `repo`, `job` |         | server     | api, microservices |
+| `IMMICH_TELEMETRY_EXCLUDE` | Do not collect these telemetries. List of `host`, `api`, `io`, `repo`, `job` |         | server     | api, microservices |
```

Comment:
You should elaborate on this to explain the behavior. I think the Prometheus doc also references the old envs and should be updated.

Reply:
Yeah, I didn't want this column to become too long. Maybe I'll just explain it in the prom docs?

Reply:
Sounds good! You should change the wording for `all` here though: it's an alternative to a list, not a normal list option.

Reply:
Good point!

Reply:
Yeah I would have list of be everything minus `all` and then either add an asterisk or inline put something like `NOTE: use all to include everything` or something like that.